endpoints/road/regions.js

```diff
@@ -0,0 +1,123 @@
+            segmentName: Road.LangtNafn[0],
+            segmentShortName: Road.StuttNafn[0],
+            segmentSignal: Road.Skilti[0].length > 0 ? Road.Skilti[0] : null,
+            conditionId: Road.IdAstand[0],
+            conditionDescription: Road.FulltAstand[0],
+            conditionShortDescription: Road.StuttAstand[0],
+            priority: parseInt(Road.Forgangur[0], 10),
+            comment: Road.Aths[0].length > 0 ? Road.Aths[0] : null,
+            date: Road.DagsKeyrtUt[0],
+            isHighlands: parseInt(Road.ErHalendi[0], 2) === 1 ? true : false,
+            colorCode: Road.Linulitur[0].length > 0 ? Road.Linulitur[0] : null,
+            conditionUpdated: Road.DagsSkrad[0],
+            surfaceCondition: Road.AstandYfirbords[0],
+          })
+      }
+      return callback(null, roads)
+    })
+  })
+}
+const getFeed = function (url, regionUrl, callback) {
+  request.get({
+    headers: { 'User-Agent': h.browser(), 'Content-Type': 'application/xml; charset=utf-8' },
+    encoding: 'utf-8',
+    url,
+  }, (error, response, body) => {
+    if (error) return callback(new Error(`${url} did not respond ${JSON.stringify(error)}`))
+    parseFeed(callback, body, regionUrl)
+  })
+}
+
+const serve = function (url, regionUrl, res, next) {
+  getFeed(url, regionUrl, (err, data) => {
+    if (err) {
+      console.error(err)
+      return next(502)
+    }
+    res.cache(1800).json({ results: data })
+  })
+}
+
+app.get('/road/:region', (req, res, next) => {
+  const regionParam = req.params.region
+  let regionUrl = null
+  if (regionParam === 'reykjavik')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/reykjavik-og-nagrenni-faerd-kort/'
+  else if (regionParam === 'west')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/vesturland-faerd-kort/'
+  else if (regionParam === 'southwest')
+    regionUrl = 'http://vegagerdin.is/ferdaupplysingar/faerd-og-vedur/sudvesturland-faerd-kort/'
+  else if (regionParam === 'westfjords')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/vestfirdir-faerd-kort/'
+  else if (regionParam === 'south')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/sudurland-faerd-kort/'
+  else if (regionParam === 'north')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/nordurland-faerd-kort/'
+  else if (regionParam === 'east')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/austurland-faerd-kort/'
+  else if (regionParam === 'northeast')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/nordausturland-faerd-kort/'
+  else if (regionParam === 'southeast')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/sudausturland-faerd-kort/'
+  else if (regionParam === 'highlands')
+    regionUrl = 'http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/midhalendid-faerd-kort/'
```

Comment:
Missing a `else` case. What happens if you region is invalid? Should return `404`.

I'd also like to see these being moved to an object. Something like

```js
// top of the file somewhere
const regionToSlug = {
  reykjavik: 'reykjavik-og-nagrenni-faerd-kort',
  west: 'vesturland-faerd-kort',
  southwest: 'sudvesturland-faerd-kort',
  // ...
}

// in request handler
const slug = regionToSlug[req.params.region]

if (!slug) {
  return next(404);
}

const regionUrl = `http://www.vegagerdin.is/ferdaupplysingar/faerd-og-vedur/${slug}/`
```